### PR TITLE
bpo-15907: Move doctest test-data files into a subdirectory of Lib/test

### PR DESCRIPTION
**Pull request by 00Kai0**
This is my first contribution for cpython :)

https://bugs.python.org/issue15907

Co-authored-by: @jaywink

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@00Kai0

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Lib/test/test_doctest.py [hunk 1]
```diff
@@ -2231,11 +2231,9 @@ def test_DocFileSuite():
        files that include examples:
 
          >>> import unittest
-         >>> suite = doctest.DocFileSuite('test_doctest.txt',
-         ...                              'test_doctest2.txt',
-         ...                              'test_doctest4.txt')
+         >>> suite = doctest.DocFileSuite('doctest_DocFileSuite_test.txt',)
```
Comment (cjerdonek) [inline Lib/test/test_doctest.py:2234]: Is there a reason this was changed?

Comment (00Kai0) [reply Lib/test/test_doctest.py:2234]: I think it should be a simple loader test. Multi file test has been done several times below.
If you think it's unreasonable, I can roll back~~

**Comment by 00Kai0** [comment]
close this pr

**Pull request closed without merging**